### PR TITLE
feat: Add new Benchmark.stub

### PR DESCRIPTION
**Pull request by jesperbeisner**
- [x] Added or updated tests
- [x] Documented user facing changes

**Changes**

Added new Benchmark.stub for right return types.

**Breaking changes**

None

### CODE REVIEW

**Comment by jesperbeisner** [comment]
I'm really not sure what's up with these failing tests. To my understanding this has nothing to do with my new extension?! :eyes:

**Comment by szepeviktor** [comment]
Those three repositories use `nunomaduro/larastan`, that causes an error.

**Review by canvural — CHANGES_REQUESTED** [review]
Hi,

This extension can be replaced by only a PHPDoc. Please put the following PHPDoc into a stub file and delete the extension. But don't modify the tests, they should still pass:
```php
/**
 * Measure a callable or array of callables over the given number of iterations.
 *
 * @template TKey
 * @param  \Closure|array<TKey, \Closure>  $benchmarkables
 * @param  int  $iterations
 * @return ($benchmarkables is \Closure ? float : array<TKey, float>)
 */
```

**Comment by canvural** [comment]
> I'm really not sure what's up with these failing tests. To my understanding this has nothing to do with my new extension?! 👀

Yeah errors were not related to this PR. And now they've been fixed. So if you rebase your branch, you'll see them passing here.

**Comment by jesperbeisner** [comment]
You learn something new every day, very cool! :muscle:

File: stubs/common/Support/Benchmark.stub [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Illuminate\Support;
+
+namespace Illuminate\Support;
```
Comment (szepeviktor) [inline stubs/common/Support/Benchmark.stub]:
```suggestion
use Closure;
```

Comment (jesperbeisner) [reply stubs/common/Support/Benchmark.stub]: Ooops 😅

**Review by CanVuralStudocu — APPROVED** [review]

**Review by canvural — APPROVED** [review]

**Comment by canvural** [comment]
Thank you!

**Pull request merged**